assets/xml/objects/object_sekihig.xml

```diff
@@ -1,16 +1,16 @@
 ﻿<Root>
     <File Name="object_sekihig" Segment="6">
-        <DList Name="object_sekihig_DL_000340" Offset="0x340" />
-        <DList Name="object_sekihig_DL_0004A8" Offset="0x4A8" />
-        <DList Name="object_sekihig_DL_0004B8" Offset="0x4B8" />
-        <DList Name="object_sekihig_DL_0005C8" Offset="0x5C8" />
+        <DList Name="gSunSongGoronSymbolBaseDL" Offset="0x340" />
+        <DList Name="gSunSongGoronSymbolDL" Offset="0x4A8" />
+        <DList Name="gSekihigGraveBaseDL" Offset="0x4B8" />
+        <DList Name="gGraveGoronSymbolDL" Offset="0x5C8" />
         <Texture Name="object_sekihig_TLUT_0005E0" OutName="tlut_0005E0" Format="rgba16" Width="4" Height="4" Offset="0x5E0" />
         <!-- <Blob Name="object_sekihig_Blob_000600" Size="0x60" Offset="0x600" /> -->
-        <Texture Name="object_sekihig_Tex_000660" OutName="tex_000660" Format="rgba16" Width="32" Height="32" Offset="0x660" />
-        <Texture Name="object_sekihig_Tex_000E60" OutName="tex_000E60" Format="rgba16" Width="32" Height="64" Offset="0xE60" />
-        <Texture Name="object_sekihig_Tex_001E60" OutName="tex_001E60" Format="ci4" Width="64" Height="64" Offset="0x1E60" />
-        <Texture Name="object_sekihig_Tex_002660" OutName="tex_002660" Format="ia4" Width="128" Height="16" Offset="0x2660" />
-        <Texture Name="object_sekihig_Tex_002A60" OutName="tex_002A60" Format="ia4" Width="128" Height="32" Offset="0x2A60" />
-        <Collision Name="object_sekihig_Colheader_0033D8" Offset="0x33D8" />
+        <Texture Name="gSekihigGraniteTex" OutName="sekihig_granite_tex" Format="rgba16" Width="32" Height="32" Offset="0x660" />
+        <Texture Name="gSekihigMarbleTex" OutName="sekihig_marble_tex" Format="rgba16" Width="32" Height="64" Offset="0xE60" />
+        <Texture Name="gSekihigGoronSymbolTex" OutName="sekihig_goron_symbol_tex" Format="ci4" Width="64" Height="64" Offset="0x1E60" />
+        <Texture Name="gSekihigStoneInscriptionTex" OutName="sekihig_stone_inscription_tex" Format="ia4" Width="128" Height="16" Offset="0x2660" />
+        <Texture Name="gSekihigStaffNotesTex" OutName="sekihig_staff_notes_tex" Format="ia4" Width="128" Height="32" Offset="0x2A60" />
+        <Collision Name="gSekihigColHeader" Offset="0x33D8" />
```

Comment:
```suggestion
        <Collision Name="gSekihigCol" Offset="0x33D8" />
```
Just Col for the suffix. Applies to the Collisions in the other objects as well.